packages/rx_bloc_cli/mason_templates/bricks/lib_auth/__brick__/lib/feature_login/views/login_page.dart

```diff
@@ -44,7 +44,8 @@ class LoginPage extends StatelessWidget {
                     ],
                   ),
                 SizedBox(height: context.designSystem.spacing.xs),
-                const GoogleLoginWidget(),{{/enable_social_logins}}
+                const GoogleLoginWidget(),{{/enable_social_logins}}{{^enable_login}}{{^enable_social_logins}}
+                Center(child: Text('No login option has been selected for the project.',textAlign: TextAlign.center,),),{{/enable_social_logins}}{{/enable_login}}
```

Comment:
Should we localize this message?

Reply:
I intentionally didn't localize it, since it's supposed to be deleted and replaced with an actual login implementation.